### PR TITLE
Introduce Aida-vm LFVM test

### PR DESCRIPTION
**Pull request by wsodsong**
This pipeline is a workaround skipping 6 unprocessable blocks.

### CODE REVIEW

File: tosca/validate-lfvm-ethereum.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            }
+        }
+    }
+
+    post {
+        always {
+            build job: '/Notifications/slack-notification', parameters: [
+                string(name: 'result', value: "${currentBuild.result}"),
+                string(name: 'name', value: "${currentBuild.fullDisplayName}"),
+                string(name: 'duration', value: "${currentBuild.duration}"),
```
Comment (simonlechner) [inline tosca/validate-lfvm-ethereum.jenkinsfile:106]: As all the other files have been updated.
```suggestion
                string(name: 'duration', value: "${currentBuild.durationString}"),
```

**Review by simonlechner — APPROVED** [review]

**Review by LuisPH3 — APPROVED** [review]

**Review by simonlechner — APPROVED** [review]

File: tosca/validate-lfvm-ethereum.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                string(name: 'duration', value: "${currentBuild.durationString}"),
+                string(name: 'url', value: "${currentBuild.absoluteUrl}"),
+                string(name: 'user', value: 'tosca')
+            ]
+        }
+    }
+}
+
+// a function for running aida-vm job for a specific range of blocks.
+def runAidaVm(first, last) {
```
Comment (LuisPH3) [inline tosca/validate-lfvm-ethereum.jenkinsfile:92]: I like this, it makes pipeline more readable.

File: tosca/validate-lfvm-ethereum.jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,103 @@
+// F17 runs LFVM on Ethereum Substate dataset.
+pipeline {
+    agent { label 'coordinator' }
+
+    environment {
```
Comment (LuisPH3) [inline tosca/validate-lfvm-ethereum.jenkinsfile:5]: nit: not needed, no go in this pipeline.

File: tosca/validate-lfvm-ethereum.jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+// a function for running aida-vm job for a specific range of blocks.
+def runAidaVm(first, last) {
+    build job: '/Tosca/Validate-LFVM', parameters: [
+        string(name: 'BlockFrom', value: "${first}"),
+        string(name: 'BlockTo', value: "${last}"),
+        string(name: 'AidaVersion', value: "${AidaVersion}"),
+        string(name: 'ToscaVersion', value: "${ToscaVersion}"),
+        string(name: 'SubstateDB', value: "/mnt/aida-ethereum-substate-db/ethereum-substate-db"),
+        string(name: 'ChainID', value: "1"),
+        string(name: 'EvmImplementation', value: "ethereum"),
```
Comment (LuisPH3) [inline tosca/validate-lfvm-ethereum.jenkinsfile:100]: I always get confused, this is using lfvm as vm, right?

**Review by LuisPH3 — APPROVED** [review]
The proposed pipeline runs in a coordinator executor and issues calls to parametrized pipelines using other agents. 
The jobs are issued concurrently and each of them executes a range of the chain history.

**Review by Lubomir-Jahn — APPROVED** [review]

**Pull request merged**